### DISCUSSION TITLE
Any coin + zil

### DISCUSSION THREAD

**Discussion by slappy6212** [RainbowMiner/RainbowMiner#2579] [Ideas]
I mined any "coin + ZIL" all the time before i Start using RainbowMiner. I was doing it with Miniz, Bzminer and lolminer. Is this feature can be add to RainbowMiner in the calculation of the revenu ?

**Comment by RainbowMiner** [comment]
You can already mine ZIL dual, but, there is no calculation of the revenue since it's a timed mining solution.
Just enter your ZIL address into Crazypool "ZIL" parameter and add Crazypool to your pools,

**Reply by slappy6212** [reply]
Great. It works.
